manifests/f/Firetrust/MailWasher/Pro/7.12.188/Firetrust.MailWasher.Pro.installer.yaml

```
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.3.2 $debug=NVS1.CRLF.7-4-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Firetrust.MailWasher.Pro
+PackageVersion: 7.12.188
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /exelang 1033 /exenoui /quiet
+  SilentWithProgress: /exelang 1033 /exenoui /quiet
```

```suggestion
  SilentWithProgress: /exelang 1033 /exenoui /quiet
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.EdgeWebView2Runtime
```
This is necessary to avoid an error 1603 on new systems